randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -2572,7 +2575,7 @@ Extra - asset_id: collision_camera_031
   > Door to EMMI Zone Exit East Access
       Can Slide
   > Tunnel to Purple EMMI Arena
-      Simple IBJ or Use Spin Boost
+      Flash Shift or Slide Jump (Beginner) or Simple IBJ or Use Spin Boost
```

Comment:
Include Slide for the Slide Jump. Also testing if you can just wall jump from a ledge grab would be worth something.

Reply:
Yes you can walljump from a ledge grab. So I put walljump (beginner) in.